packages/grant-explorer/src/features/discovery/LandingPage.tsx

```diff
@@ -36,14 +36,21 @@ const LandingPage = () => {
         action={<ViewAllLink to="#">View all</ViewAllLink>}
       >
         <div className="grid md:grid-cols-3 gap-x-6">
-          {activeRounds.data?.slice(0, 4).map((round, i) => (
-            <div
-              key={round.id}
-              className={`${i % 3 === 0 ? "md:col-span-2" : ""}`}
-            >
-              <RoundCard round={round} />
-            </div>
-          ))}
+          {activeRounds.data
+            ?.filter(
+              (round) =>
+                round?.projects?.length !== undefined &&
+                round.projects.length > 0
+            )
+            .slice(0, 4)
+            .map((round, i) => (
+              <div
+                key={round.id}
+                className={`${i % 3 === 0 ? "md:col-span-2" : ""}`}
+              >
+                <RoundCard round={round} />
+              </div>
+            ))}
```

Comment:
Is it possible to do this on a query level in the graph? If not, move this logic to the hook useActiveRounds:

```ts
export function useActiveRounds() {
  const [currentTimestamp] = useState(createTimestamp());
  const [futureTimestamp] = useState(createTimestamp(ONE_YEAR * 10));
  const query = useRounds({
    orderBy: "roundEndTime",
    orderDirection: "desc",
    where: {
      // Round must have started and not ended yet
      roundStartTime_lt: currentTimestamp,
      roundEndTime_gt: currentTimestamp,
      roundEndTime_lt: futureTimestamp,
    },
  });
  return {
    ...query,
    data: query.data?.filter(round => round.projects?.length)
  }
}
```